governance/polls/MIP10c14-SP7: Appoint DeFi Saver as Oracle Light Feed.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Appoint DeFi Saver as Oracle Light Feed (MIP10c14-SP7) - April 5, 2021
+summary: Signal your support or opposition for appointing DeFi Saver as Oracle Light Feed
+discussion_link: https://forum.makerdao.com/t/mip10c14-sp7-proposal-appoint-defi-saver-as-a-light-feed/7207
+vote_type: Plurality Voting
+categories:
+   - Oracles
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Appoint DeFi Saver as Oracle Light Feed (MIP10c14-SP7) - April 5, 2021
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of DeFi Saver. This Governance Poll will be active for three days beginning on Monday, April 5 at 16:00 UTC.
+
+Note: This subproposal has been submitted under the informal weekly cycle rather than the MIP3 Governance Cycle due to inconsistencies between MIP10 and MIP3 and to avoid delaying progress in the oracles domain until MIP10 can be amended.
+
+**This is a binary vote. In a binary vote, you may vote for a single option. You should vote for the option which you prefer. Alternatively, if you would accept either of the two options and have no preference between them, you should abstain.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to appointing DeFi Saver as Oracle Light Feed.
+
+Please review the application [thread](https://forum.makerdao.com/t/mip10c14-sp7-proposal-appoint-defi-saver-as-a-light-feed/7207) to inform your position on this issue before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option, then the following actions will be taken:**
+* This change will be included in a Friday Executive Vote within the next 30 days, absent external factors.
+* If the Executive Vote passes, then this change will become active in the Maker Protocol after the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay) has expired.
+
+**If the votes for the 'No' option equal or exceed the votes for the 'Yes' option, then no further action will be taken at this time.**
+
+---
+
+## Resources
```

Comment:
Realized this is worded slightly differently than the standard template. I think it works tho, the Governance Risk Framework one was the only line that stuck out and encouraged me to compare the two